conans/client/remote_registry.py

```diff
@@ -83,68 +85,88 @@ def __init__(self, filename, lockfile, output):
         self._filename = filename
         self._lockfile = lockfile
         self._output = output
-        self._remotes = None
+        # Cache json contents
+        self._contents = ""
```

Comment:
Dangerous. The contents can be changed by a concurrent process.

Reply:
Ok, I'll remove the cache